### PR TITLE
Issue #9: remove dependance on libraries module, bundle colorbox in c…

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop-contrib/colorbox/issues/9
- bundles colorbox library & removes any reference to libraries project.
- UX cleanup of admin settings form
- Backdrop coding standards cleanup in module
- removed unnecessary files (.make and .variables)
- cleaned up README.md to match required standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- Visit the configuration page under Administration > Configuration > Media >
+  Colorbox (admin/config/media/colorbox) and configure as necessary.
+
+
+Documentation
+-------------
+
+Drupal Documentation may be found here: https://www.drupal.org/node/1971564
+
+Additionl Documentation is located in the Wiki:
```
Comment (klonos) [inline README.md]: typo: "Addition**a**l"

**Review by klonos — CHANGES_REQUESTED** [review]

File: colorbox.admin.inc [hunk 2]
```diff
@@ -113,31 +92,30 @@ function colorbox_admin_settings() {
...
-    '#title' => t('Next'),
+    '#title' => t('Text for the next button'),
     '#default_value' => config_get('colorbox.settings', 'colorbox_text_next'),
     '#size' => 30,
-    '#description' => t('Text for the next button in a shared relation group.'),
   );
   $form['colorbox_custom_settings']['colorbox_text_close'] = array(
     '#type' => 'textfield',
-    '#title' => t('Close'),
+    '#title' => t('Text for the close button.'),
```
Comment (klonos) [inline colorbox.admin.inc]: Remove period there.

File: colorbox.admin.inc [hunk 3]
```diff
@@ -147,101 +125,105 @@ function colorbox_admin_settings() {
...
+    '#states' => array(
+      'visible' => array(
+        ':input[name="colorbox_slideshow"]' => array('value' => '1'),
+      ),
+    ),
+  );
+  $form['colorbox_slideshow_settings']['colorbox_slideshowauto'] = array(
     '#type' => 'checkbox',
-    '#title' => t('Slideshow autostart'),
+    '#title' => t('Automatically start to play the slideshow.'),
```
Comment (klonos) [inline colorbox.admin.inc]: Description/help text must have periods. Checkbox text and labels should not AFAIK.

File: colorbox.admin.inc [hunk 4]
```diff
@@ -147,101 +125,105 @@ function colorbox_admin_settings() {
...
+    '#field_suffix' => t('milliseconds'),
     '#options' => backdrop_map_assoc(array(1000, 1500, 2000, 2500, 3000, 3500, 4000, 4500, 5000, 5500, 6000)),
     '#default_value' => config_get('colorbox.settings', 'colorbox_slideshowspeed'),
-    '#description' => t('Sets the speed of the slideshow, in milliseconds.'),
   );
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings']['colorbox_text_start'] = array(
+  $form['colorbox_slideshow_settings']['colorbox_text_start'] = array(
     '#type' => 'textfield',
-    '#title' => t('Start slideshow'),
+    '#title' => t('Start button text:'),
```
Comment (klonos) [inline colorbox.admin.inc]: Nice moving the help text to the label, but no colon.

File: colorbox.admin.inc [hunk 5]
```diff
@@ -147,101 +125,105 @@ function colorbox_admin_settings() {
...
     '#default_value' => config_get('colorbox.settings', 'colorbox_text_start'),
     '#size' => 30,
-    '#description' => t('Text for the slideshow start button.'),
   );
-  $form['colorbox_custom_settings']['colorbox_slideshow_settings']['colorbox_text_stop'] = array(
+  $form['colorbox_slideshow_settings']['colorbox_text_stop'] = array(
     '#type' => 'textfield',
-    '#title' => t('Stop slideshow'),
+    '#title' => t('Stop button text:'),
```
Comment (klonos) [inline colorbox.admin.inc]: ...again, no colon.

**Review by klonos — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jenlampton) [reply README.md]: fixed :)

File: colorbox.admin.inc (hunk 2 above)
Comment (jenlampton) [reply colorbox.admin.inc]: fixed :)

File: colorbox.admin.inc (hunk 3 above)
Comment (jenlampton) [reply colorbox.admin.inc]: fixed :)

File: colorbox.admin.inc (hunk 4 above)
Comment (jenlampton) [reply colorbox.admin.inc]: fixed

File: colorbox.admin.inc (hunk 5 above)
Comment (jenlampton) [reply colorbox.admin.inc]: fixed

**Comment by jenlampton** [comment]
Thanks for the review @klonos! :)

**Review by klonos — APPROVED** [review]

**Comment by klonos** [comment]
Not a very thorough one as I'm 💤 and only checked user-facing text, so could use a second pair of eyes.

**Pull request merged**